org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/ILanguageServerInfoProvider.java

```diff
@@ -0,0 +1,15 @@
+package org.eclipse.jdt.ls.debug.adapter;
+
+public interface ILanguageServerInfoProvider {
+    String getFullyQualifiedName(String sourceFilePath, int line, int column);
+
+    String getSourceFilePath(String fullyQualifiedName, String projectName);
+
+    String getURI(String fullyQualifiedName, String projectName);
```

Comment:
Actually `getSourceFilePath` and `getURI` do the same thing. Just need one API `getURI`.